ui/packages/catalog-ui-search/src/main/webapp/component/alert-settings/alert-settings.less

```diff
@@ -2,28 +2,26 @@
   .customElement;
 
   overflow: auto;
-  padding: @minimumSpacing @minimumSpacing;
-  font-size: @largeFontSize;
+  font-size: @minimumFontSize;
+  padding: @minimumSpacing;
 
   .editor-properties {
     overflow: auto;
     max-height: ~"calc(100% - @{minimumButtonSize} - 2*@{minimumSpacing})";
-    text-align: center;
+    text-align: left;
 
     .property-label {
-      text-align: center;
+      .is-bold();
     }
   }
 
   .property-expiration {
     display: none;
   }
 
-  .editor-properties {
-    .property-label {
-      height: @minimumButtonSize;
-      line-height: @minimumButtonSize;
-      .is-bold();
+  .property-persistance {
+    > .property-region {
+      margin: -10px 0 20px 0;
```

Comment:
Why are we using negative margin here?

Reply:
To fix the spacing between the "persist across sessions" title and the intrigue-radio.
Intrigue-radio being inline-block seems to cause that spacing and I didn't want to change that directly because I didnt want it to affect other spots where that intrigue-radio is being used